### ISSUE TITLE
Tensorflow Model Format and Checkpoints

### ISSUE THREAD

**Issue by sjawhar** [sjawhar/no-wander#7]
## Purpose
* Save model in Tensorflow format so it can be picked up automatically by Gradient
* Use checkpoints to save model every few epochs, so training can be terminated early if needed
* Explore integrating with Gradient Metrics

## Possible Solutions
* https://keras.io/callbacks/#example-model-checkpoints
